packages/component-ui/src/icon-button/icon-button.tsx

```diff
@@ -47,21 +55,29 @@ export function IconButton({
       'h-10 w-10': size === 'large' && !isNoPadding,
       'inline-flex': props.isAnchor,
       'pointer-events-none': isDisabled,
+      [buttonColors[variant].selected]: isSelected,
+      [buttonColors[variant].base]: !isSelected,
     },
   );
 
   const iconSize = size === 'small' ? 'small' : 'medium';
 
   if (props.isAnchor === true) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { isAnchor, ...buttonProps } = props;
```

Comment:
```suggestion
    const buttonProps = Object.fromEntries(
      Object.entries(props).filter(([key]) => key !== 'isAnchor')
    ) as Omit<typeof props, 'isAnchor'>;
```

で`no-unused-vars`を使わずに、`isAnchor`以外のpropsデータを抽出できます。